components/post_view/post_body/index.js

```diff
@@ -23,7 +23,8 @@ function mapStateToProps(state, ownProps) {
         ...ownProps,
         parentPost,
         parentPostUser,
-        previewCollapsed: get(state, Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.COLLAPSE_DISPLAY, 'false')
+        previewCollapsed: get(state, Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.COLLAPSE_DISPLAY, 'false'),
+        previewEnabled: get(state, Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.LINK_PREVIEW_DISPLAY, Preferences.LINK_PREVIEW_DISPLAY_DEFAULT) === 'true'
```

Comment:
I don't think you need to change it now since most places just use this selector, but there's a getBool selector you can use so that you don't need to do the string comparison yourself